### DISCUSSION TITLE
BetterStack/logtail integration - how to pick what types of logs are sent to betterstack

### DISCUSSION THREAD

**Discussion by ivanjd** [vercel/community#4493] [Help] [answered]
### Summary

I am currently using the logtail integration from the marketplace and it works great, but out of the box it is sending too much logs. Is there a way to limit it. For example, I do not want to see the logs for loading images like the one here:
![image](https://github.com/vercel/community/assets/8286519/d8a74ca2-4161-4785-82d1-11a7194848fb)

Basically, I need the logs for the serverless functions only. 
Thanks.

### Steps to Reproduce

- enable the vercel betterstack integration

**Comment by amyegan** [comment, marked as answer]
Hi @ivanjd. You can find information about how to configure Better Stack (Logtail) in their docs.

https://betterstack.com/docs/logs/vercel/
